packages/documentation/copy/en/javascript/JSDoc Reference.md

```diff
@@ -463,25 +593,55 @@ function callbackForLater(e) {
 }
 ```
 
-## `@extends`
+# Documentation
+
+## `@deprecated`
+<div id="[deprecated-comments]"></div>
 
-When JavaScript classes extend a generic base class, there is nowhere to specify what the type parameter should be. The `@extends` tag provides a place for that type parameter:
+When a function, method, or property is deprecated you can let users know by marking it with a `/** @deprecated */` JSDoc comment. That information is surfaced in completion lists and as a suggestion diagnostic that editors can handle specially. In an editor like VS Code, deprecated values are typically displayed in a strike-through style ~~like this~~.
 
 ```js twoslash
-/**
- * @template T
- * @extends {Set<T>}
- */
-class SortableSet extends Set {
-  // ...
+// @noErrors
+/** @deprecated */
+const apiV1 = {};
+const apiV2 = {};
+
+apiV;
+// ^|
+
+
+```
+
+## `@see`
+
+`@see` lets you link to other names in your program:
```

Comment:
I wonder if 'names` here could be expanded upon in some sense, as the term feels a bit ambiguous

Reply:
identifiers? entities? types'n'values?